test/loader.spec.js

```diff
@@ -208,19 +208,6 @@ describe('loader', () => {
 					{}
 				)
 			);
-
-			it(
-				'should configure emitCss=true',
```

Comment:
how come this test was removed?

other than that, this PR looks good to me

Reply:
the test fails on this branch for some reason with the `expect(err).not.to.exist` line

Reply:
It seems like something we should probably fix? I think I'd be happy to merge this after that

Reply:
Here's the stack trace of the underlying error:
```
CompileError [ValidationError]: A component cannot have a default export
    at error$1 (/home/smit/svelte-loader/node_modules/svelte/compiler.js:15722:20)
    at Component.error (/home/smit/svelte-loader/node_modules/svelte/compiler.js:27639:10)
    at Component.extract_exports (/home/smit/svelte-loader/node_modules/svelte/compiler.js:27671:19)
    at Object.enter (/home/smit/svelte-loader/node_modules/svelte/compiler.js:27931:52)
    at visit (/home/smit/svelte-loader/node_modules/svelte/compiler.js:5256:11)
    at visit (/home/smit/svelte-loader/node_modules/svelte/compiler.js:5288:13)
    at walk (/home/smit/svelte-loader/node_modules/svelte/compiler.js:5207:10)
    at Component.post_template_walk (/home/smit/svelte-loader/node_modules/svelte/compiler.js:27916:10)
    at Component.walk_instance_js_post_template (/home/smit/svelte-loader/node_modules/svelte/compiler.js:27898:15)
    at new Component (/home/smit/svelte-loader/node_modules/svelte/compiler.js:27454:15)
    at compile (/home/smit/svelte-loader/node_modules/svelte/compiler.js:28637:24)
```

Reply:
Debugging: in `const resource = posixify(this.resource)`, `this.resource` is `undefined`

Reply:
@non25: I've figured out the issue, https://github.com/non25/svelte-loader/pull/2 re-adds the test in a way that makes it pass.

Reply:
I can't fix it and wrote about it in the post. Would be nice if somebody can help me with that.